processed 19 tasks

init:
A: object(0,0)

task 1, lines 9-112:
//# publish
created: object(1,0)
mutated: object(0,1)
gas summary: computation_cost: 1000000, storage_cost: 12638800,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2, line 114:
//# run Test::M::prim_vec_len --sender A --args vector[7,42]
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 980400,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3, lines 116-120:
//# run Test::M::mint --sender A --args 42
created: object(3,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2257200,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 4, line 122:
//# run Test::M::obj_vec_destroy --sender A --args vector[object(3,0)]
mutated: object(0,0)
deleted: object(3,0)
gas summary: computation_cost: 1000000, storage_cost: 980400,  storage_rebate: 2257200, non_refundable_storage_fee: 0

task 5, line 124:
//# run Test::M::mint --sender A --args 42
created: object(5,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2257200,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 6, line 126:
//# run Test::M::mint_child --sender A --args 42 object(5,0)
created: object(6,0), object(6,1)
mutated: object(0,0), object(5,0)
gas summary: computation_cost: 1000000, storage_cost: 5981200,  storage_rebate: 2257200, non_refundable_storage_fee: 0

task 7, lines 128-132:
//# run Test::M::child_access --sender A --args object(5,0) vector[object(6,0)]
Error: Error checking transaction input objects: InvalidChildObjectArgument { child_id: object(6,0), parent_id: object(6,1) }

task 8, line 134:
//# run Test::M::mint_another --sender A --args 42
created: object(8,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2310400,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 9, lines 136-139:
//# run Test::M::obj_vec_destroy --sender A --args vector[object(8,0)]
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(1) } }

task 10, line 141:
//# run Test::M::mint_another --sender A --args 42
created: object(10,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2310400,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 11, line 143:
//# run Test::M::mint --sender A --args 42
created: object(11,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2257200,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 12, lines 145-148:
//# run Test::M::two_obj_vec_destroy --sender A --args vector[object(10,0),object(11,0)]
Error: Transaction Effects Status: Invalid command argument at 1. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 1, kind: TypeMismatch }, source: None, command: Some(0) } }

task 13, line 150:
//# run Test::M::mint_shared --sender A --args 42
created: object(13,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2257200,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 14, lines 152-155:
//# run Test::M::obj_vec_destroy --sender A --args vector[object(13,0)]
mutated: object(0,0)
deleted: object(13,0)
gas summary: computation_cost: 1000000, storage_cost: 980400,  storage_rebate: 2257200, non_refundable_storage_fee: 0

task 15, line 157:
//# run Test::M::mint --sender A --args 42
created: object(15,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2257200,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 16, lines 159-162:
//# run Test::M::same_objects --sender A --args object(15,0) vector[object(15,0)]
Error: Transaction Effects Status: Invalid command argument at 0. Invalid usage of value. Mutably borrowed values require unique usage. Immutably borrowed values cannot be taken or borrowed mutably. Taken values cannot be used again.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidValueUsage }, source: None, command: Some(1) } }

task 17, line 164:
//# run Test::M::mint --sender A --args 42
created: object(17,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2257200,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 18, line 166:
//# run Test::M::same_objects_ref --sender A --args object(17,0) vector[object(17,0)]
Error: Transaction Effects Status: Invalid command argument at 0. Invalid usage of value. Mutably borrowed values require unique usage. Immutably borrowed values cannot be taken or borrowed mutably. Taken values cannot be used again.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidValueUsage }, source: None, command: Some(1) } }
